src/gui/socketapi/socketuploadjob.cpp

```diff
@@ -73,6 +73,10 @@ void SocketUploadJob::prepareTag(const AccountPtr &account)
 
 void SocketUploadJob::start()
 {
+    fail("needs porting");
```

Comment:
Note for self: PR is still WIP.

Reply:
nope I created another issue for this, this code is unused an appears to be broken for a long time now.
https://github.com/owncloud/client/issues/9585
Marked as p2 so we don't move it to 2.12 or something like that.